### ISSUE TITLE
JSON-Syntax-Error when compiling "faq_de.json" and "faq.json"

### ISSUE THREAD

**Issue by c-mahn** [corona-warn-app/cwa-website#67]
In the above mentioned files a few closing parentheses are missing, which cause "gulp build" to fail.
```
[christopher@ArchDesktop cwa-website]$ gulp build
[11:41:39] Using gulpfile ~/Schreibtisch/cwa-website/gulpfile.js
[11:41:39] Starting 'build'...
[11:41:39] Starting 'pages'...
[11:41:39] Starting 'javascript'...
[11:41:39] Starting 'images'...
[11:41:39] Starting 'copy'...
[11:41:40] 'pages' errored after 89 ms
[11:41:40] SyntaxError: /home/christopher/Schreibtisch/cwa-website/src/data/faq_de.json: Unexpected token 
 in JSON at position 21610
    at parse (<anonymous>)
    at Object.Module._extensions..json (internal/modules/cjs/loader.js:1234:22)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Panini.module.exports [as loadData] (/home/christopher/Schreibtisch/cwa-website/node_modules/panini/lib/loadData.js:21:14)
    at Panini.module.exports [as refresh] (/home/christopher/Schreibtisch/cwa-website/node_modules/panini/lib/refresh.js:10:8)
    at module.exports (/home/christopher/Schreibtisch/cwa-website/node_modules/panini/index.js:41:12)
    at pages (/home/christopher/Schreibtisch/cwa-website/gulpfile.js:60:7)
[11:41:40] 'build' errored after 91 ms
[11:41:40] The following tasks did not complete: javascript, images, copy
[11:41:40] Did you forget to signal async completion?
[christopher@ArchDesktop cwa-website]$ 
```

**Comment by c-mahn** [comment]
I have the files with the fix ready, but I don't know how to commit the changes.

**Comment by OlMue** [comment]
@c-mahn Thanks! There's a related pull request: [https://github.com/corona-warn-app/cwa-website/pull/66](https://github.com/corona-warn-app/cwa-website/pull/66). It hasn't been merged yet, so feel free to take a look and comment.

**Comment by c-mahn** [comment]
Issue solved by merge request #66 .
